docs/building_on_linux.md

```diff
@@ -30,14 +30,25 @@ schismtracker directory and running:
 
 ## Building Schism Tracker
 
-To build Schism Tracker, you should set up a build-directory. From the
-schismtracker directory:
+To build Schism Tracker, you should set up a build-directory and compile from
+there. From the schismtracker directory:
 
-    mkdir -p build && cd build && ../configure && make
+    mkdir -p build && cd build && ../configure && \
+    make -j $(nproc || sysctl -n hw.ncpu || echo 2)
```

Comment:
I'd rather keep things simple here and have the basic build instructions just use `make` without any optimization flags, since multithreading flags are mentioned below for anyone who wants them.

Reply:
done